index.html

```diff
@@ -3404,9 +3404,8 @@ <h2>Definition of Roles</h2>
 			<rdef>generic</rdef>
 			<div class="role-description">
 				<p>A nameless container <a>element</a> that has no semantic meaning on its own.</p>
-				<p>Contrast with <rref>group</rref>, which semantically groups its descendants in a named container.</p>
-				<p>A <code>generic</code> can provide a limited number of accessible states and properties for its descendants, such as <pref>aria-live</pref> attributes. This differentiates it from the <rref>presentation</rref> role.</p>
-				<p>The <code>generic</code> role is intended for implementors of User Agents. Authors SHOULD NOT use this role in content.</p>
+				<p>The <code>generic</code> role is intended for implementors of User Agents. Authors SHOULD NOT use this role in content. Authors MAY use <rref>presentation</rref> or <rref>none</rref> to remove implicit accessibility semantics, or a semantic container role, such as <rref>group</rref>, to semantically group descendants in a named container.</p>
+				<p>Like an element with role <rref>presentation</rref>, a <code>generic</code> can provide a limited number of accessible states and properties for its descendants, such as <pref>aria-live</pref> attributes. However, unlike an element with role <rref>presentation</rref>, <code>generic</code> elements are exposed in <a>accessibility APIs</a> so that assistive technologies can gather certain properties such as layout and bounds.</p>
```

Comment:
Nit: I personally don't think we want to treat "generic" as a noun (i.e. in the first sentence).

Reply:
changed "a generic" to "an element with role generic"